.github/workflows/main-build.yml

```diff
@@ -14,14 +14,16 @@ jobs:
         with:
           java-version: 14
       - uses: gradle/wrapper-validation-action@v1
-      - name: Login to GitHub Package Registry
-        run: echo ${{ secrets.CONTAINER_REGISTRY_TOKEN }} | docker login ghcr.io -u anuraaga --password-stdin
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-west-2
       - name: Login to ECR
         uses: docker/login-action@v1
         with:
-          registry: ${{ secrets.AWS_ACCOUNT }}.dkr.ecr.us-west-2.amazonaws.com
-          username: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          password: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          registry: public.ecr.ews
```

Comment:
Shouldn't this be `public.ecr.aws`?

Reply:
It is, isn't it?

Reply:
I mean it's `ews` currently

Reply:
Doh thought I copy-pasted this from somewhere but apparently not, thanks for seeing it!